snuba/attribution/__init__.py

```diff
@@ -1,31 +1,24 @@
 from __future__ import annotations
 
-from snuba import environment, state
+from datetime import datetime
+
+from snuba import environment
 from snuba.utils.metrics.wrapper import MetricsWrapper
 
 from .appid import AppID
-from .default import APPIDS as DEFAULT_APPIDS
 
 metrics = MetricsWrapper(environment.metrics, "snuba.attribution")
 
 
-APPIDS: dict[str, AppID] = {}
-DEFAULT_APPID = AppID.from_dict(DEFAULT_APPIDS[0])
-INVALID_APPID = AppID.from_dict(DEFAULT_APPIDS[1])
-for obj in DEFAULT_APPIDS:
-    app_id = AppID.from_dict(obj)
-    APPIDS[app_id.key] = app_id
+DEFAULT_APPID = AppID("default", "evanh", datetime(2022, 3, 24))
```

Comment:
Nit: You can probably set the created by field to "snuba" and not have your name in the default cases

Reply:
But then people won't know who created them! ;)